native/chat/settings/thread-settings.react.js

```diff
@@ -382,6 +388,11 @@ class ThreadSettings extends React.PureComponent<Props, State> {
       key: 'pushNotifs',
       threadInfo,
     });
+    listData.push({
```

Comment:
This class has changed since you put the PR up. (The `listData` is now defined in a selector instead of in the `render` function.)

Can you rebase your changes against master?